requirements.txt

```diff
@@ -1,7 +1,7 @@
 fonttools==3.9.2
 cu2qu==1.1.1
 glyphsLib==1.6.0
-ufo2ft==0.4.2
+ufo2ft==0.4.3
```

Comment:
I never pushed a 0.4.3 tag so there's no package with that version on pypi. Sorry for the misunderstanding.
The next ufo2ft version will be 0.5.0. It will be live only after we cut a release and push the tag.

Note that in this case, it is not sufficient to bump the ufo2ft version in "requirements.txt". You also need to manually bump the *minimum required version* of ufo2ft in the setup.py `install_requires`.
Otherwise, if one attempts to `pip install fontmake` in an environment which already contains a version of "ufo2ft>=0.4.2", the latter will not be updated automatically, since pip will think that the requirement is already satisfied.